packages/select/src/components/query-list/queryList.tsx

```diff
@@ -248,7 +268,13 @@ export class QueryList<T> extends React.Component<IQueryListProps<T>, IQueryList
     private getActiveIndex(items = this.state.filteredItems) {
         const { activeItem } = this.state;
         // NOTE: this operation is O(n) so it should be avoided in render(). safe for events though.
-        return activeItem == null ? -1 : items.indexOf(activeItem);
+        // TODO: Why wasn't I allowed to use Array.prototype.findIndex?
```

Comment:
I wanted to simply use items.findIndex(...) here, but it's not available with the current tsc configuration (and I assume a polyfill is not included). Is there an equivalent utility method in this project I should use? Is this ad-hoc implementation OK?

Reply:
see https://blueprintjs.com/docs/#blueprint/getting-started.language-features. `findIndex` is not in our JS environment. please remove TODO, you did the right thing.

Reply:
TODO deleted